conference/awards.md

```diff
@@ -0,0 +1,19 @@
+| 2022 | People's Choice Award: Best Poster | Explicating Revisions – How to mediate Beethoven's redaction instructions                                                                                   | Kepper, Johannes, Jan-Peter Voigt, Susanne Cox, Ran Mo, Elisa Novara, Richard York Sänger, Agnes Seipelt                                                                                                                       |
+| 2022 | People's Choice Award: Best Poster | Towards MerMEId 2.0                                                                                                                                         | Stadler, Peter, Omar Siam, Margrethe Støkken Bue, Clemens Gubsch, David Lewis, Kevin Page, Axel Teich Geertinger, Sigfrid Lundberg, Joshua Neumann, Daniel Jettka, Anastasia Wawilow, Zsofia Karolina Abraham, Daniel Schopper |
+| 2022 | People's Choice Award: Best Poster | Encoding and Analyzing a New Corpus of Popular Songs                                                                                                        | Reymore, Lindsey, Matthew Zeller, Leigh VanHandel, Ben Duinker, Nicholas Shea, Christopher William White, Jeremy Tatar, Jade Roth, Nicole Biamonte                                                                             |
+| 2021 | Best Paper                         | Alleviating the Last Mile of Encoding: The mei-friend Package for the Atom Text Editor                                                                      | Goebl, W.; and M. Weigl, D.                                                                                                                                                                                                    |
+| 2021 | Best Poster                        | The OpenScore Lieder Corpus                                                                                                                                 | Gotham, M. R. H.; and Jonas, P.                                                                                                                                                                                                |
+| 2019 | Best Paper                         | Application of Music-Encoding Technologies to Guatemalan Choirbooks, Facilitating Preservation and Musicological Studies of the Colonial Repertoire         | Thomae, Martha, Julie E. Cumming, Ichiro Fujinaga                                                                                                                                                                              |
```

Comment:
Would it make sense to have an entry for 2020 indicating that there were no awards given? (To signalize that it is not an oversight)

Reply:
I will update the file and add it here.

Reply:
[awards.md](https://github.com/music-encoding/music-encoding.github.io/files/9607831/awards.md)

Reply:
Thank you